<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkDriveDialog</class>
 <widget class="QDialog" name="NetworkDriveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Dokany Network Drive Example</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="serverLabel">
     <property name="text">
      <string>Server:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="serverLineEdit">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QPushButton" name="connectPushButton">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="driveLetterLabel">
     <property name="text">
      <string>Drive Letter:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="driveLetterComboBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>H</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>I</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>J</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>L</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>N</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>O</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>P</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Q</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>S</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>T</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>U</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>W</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Z</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="logPlainTextEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
